app/controllers/families_controller.rb

```diff
@@ -17,12 +17,6 @@ def create
     respond_with @family, location: -> { families_path }
   end
 
-  def destroy
-    family.delete
-
-    respond_with @family, location: -> { families_path }
-  end
-
```

Comment:
This does not belong in this PR. Did you rebase with master to get the latest changes?

Reply:
I think I deleted it because I realised that didn't belong to this PR.. :sweat_smile:

Reply:
👍